node_modules/gh-admins/lib/rest.js

```diff
@@ -53,7 +55,13 @@ GrassHopper.globalAdminRouter.on('get', '/api/admins', function(req, res) {
  * @Return      {Me}                                            The current global administrator
  */
 GrassHopper.globalAdminRouter.on('get', '/api/me', function(req, res) {
-    res.status(200).send(req.ctx.user);
+    var me = {
```

Comment:
I would still prefer to have this in an API function. That would make it easier to return the me object when logging in or when updating the current user profile.